### DISCUSSION TITLE
antsMultivariateTemplateConstruction2 generates template considerably smaller then input template

### DISCUSSION THREAD

**Discussion by MRIDude** [ANTsX/ANTs#1770] [General]
I am generating a multivariate template using a t1 and T2 from 70 participants. I first compute a linear template using antsMultivariateTemplateConstruction2. Then register I the template using affine transform to MNI152 space. Then give the registered affine template back to  antsMultivariateTemplateConstruction2 to generate a nonlinear template. I noticed  that the finally output was considerably smaller then the registered affine template.  I was thinking it may be a drift over 4 iteration. So i registered the the non-linear target to MNI space again and ran it through antsMultivariateTemplateConstruction2 for one more iteration. The final target is still shrunk.  Everything that goes in to antsMultivariateTemplateConstruction2 look fine. just the output is shrunk. Below is the command I used to generate the nonlinear target. Do any of you know why this is happening?

`antsMultivariateTemplateConstruction2.sh -d 3 
                                         -i 1 -k 2 \
                                         -f 8x4x2x1x1 \ 
                                         -y 0 \
                                         -s 3x2x1x0x0vox \
                                         -q 200x100x70x20x20 \
                                         -t SyN  \
                                         -m CC \
                                         -c 0 \
                                         -y 0 \
                                         -n 0 \
                                         -z PIBNLtemplate0_on_MNI152lin_T1_0p9mm.nii.gz -z PIBNLtemplate1_on_MNI152lin_T1_0p9mm.nii.gz \
                                         -o PIBNL_NL_MNI152 \
                                         $ProjectDIR/Image_BC.csv || exit -1`

**Comment by cookpa** [comment]
I would guess it's some kind of misunderstanding in the image headers
